### PR TITLE
Fixed issue with views/web.config

### PR DESCRIPTION
**Pull request by rasmuseeg**
Was using incorrect version of mvc

### CODE REVIEW

**Comment by rasmuseeg** [comment]
This should have been request to release/5

File: Bonobo.Git.Server/Views/Web.config [hunk 1]
```diff
@@ -2,71 +2,77 @@
 
 <configuration>
   <configSections>
-	<sectionGroup name="system.web.webPages.razor" type="System.Web.WebPages.Razor.Configuration.RazorWebSectionGroup, System.Web.WebPages.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35">
```
Comment (RedX2501) [inline Bonobo.Git.Server/Views/Web.config]: Please do not fix the indentation of the file if there are also functional changes in the file.

**Comment by willdean** [comment]
@rasmuseeg Not sure if you're still trying to submit this, but if you are, can you squash all the commits so that it's easy to see what's changed?

**Comment by rasmuseeg** [comment]
Hi m8, just i'm 300 commits behind. Was trying to push to master, but ended up with a version of release/5.

I'll redo this pull, and return.

**Comment by willdean** [comment]
@rasmuseeg I think when you fetch, you'll find the MVC version issue was fixed about 2 months ago...

**Pull request closed without merging**